src/gt4py/next/otf/binding/pybind.py

```diff
@@ -108,13 +123,7 @@ class BindingCodeGenerator(TemplatedGenerator):
     )
 
     def visit_FunctionParameter(self, param: FunctionParameter):
-        if isinstance(param.type_, ts.FieldType):
-            type_str = "pybind11::buffer"
-        elif isinstance(param.type_, ts.ScalarType):
-            type_str = cpp_interface.render_scalar_type(param.type_)
-        else:
-            raise ValueError(f"Type '{param.type_}' is not supported in pybind11 interfaces.")
-        return f"{type_str} {param.name}"
+        return f"{_type_string(param.type_)} {param.name}"
```

Comment:
Not important, but this could just be a format template instead of a visit method right?

Reply:
fixed